### PR TITLE
Python update

### PR DESCRIPTION
**Pull request by pelahi**
These updates were driven by the fact that the recipes for hpc-python were quite old and had a number of security holes. 

The updates are to remove some critical sercurity holes and the related docker images on quay.io have been removed.

### CODE REVIEW

File: python/hpc-python-hdf5mpi/Dockerfile
Comment (dipietrantonio) [inline python/hpc-python-hdf5mpi/Dockerfile:1]: I wonder whether we should mark Intel containers as deprecated (we do not want to maintain containers users can't use on our systems). I am wondering the same for CUDA, even though we might get NVIDIA nodes in the near future.

**Comment by Elwell** [comment]
I'd vigorously mark anything we don't actively maintain and patch as
deprecated or unmaintained - that way anyone else using them should realise
the risks they're taking

**Comment by pelahi** [comment]
@dipietrantonio sorry for reply to this comment so late but I don't understand your comment. The recipes are marked as deprecated. The containers I just removed as they have critical security holes. What else would you like to happen?

**Comment by dipietrantonio** [comment]
@pelahi what I meant is that we should remove them not only because of security, but also because we do not have an intel (user facing) machine anymore. Maybe let's create a tag for future reference. In other words, I worried that we would fix security holes and maintain the containers that won't be used on Setonix.

**Comment by pelahi** [comment]
Hi @dipietrantonio I did move the intel recipes to deprecated. I deleted the containers but let the repositories as empty. Did you want me to remove the repos entirely?

**Review by dipietrantonio — APPROVED** [review]
I had not noticed the "Deprecated" directory. All good!

**Pull request merged**